blockchain/projects.go

```diff
@@ -134,50 +133,21 @@ func (p *Projects) ExecuteScript(execution model.NewScriptExecution) (*model.Scr
 	return exe, nil
 }
 
-// GetAccount by the address along with its storage information.
-func (p *Projects) GetAccount(projectID uuid.UUID, address model.Address) (*model.Account, error) {
+// CreateInitialAccounts returns the number of accounts that were created
+func (p *Projects) CreateInitialAccounts(projectID uuid.UUID) (*int, error) {
 	p.mutex.load(projectID).Lock()
 	defer p.mutex.remove(projectID).Unlock()
-	em, err := p.load(projectID)
-	if err != nil {
-		return nil, err
-	}
-
-	return p.getAccount(em, projectID, address)
-}
-
-func (p *Projects) GetAccounts(projectID uuid.UUID, addresses []model.Address) ([]*model.Account, error) {
```

Comment:
Accounts shouldn't be removed completely, they should be kept to include `storage`, `address` and optionally `contracts`